doc/datavolumes.md

```diff
@@ -39,6 +39,32 @@ spec:
 [Get example](../manifests/example/example-import-dv.yaml)
 [Get secret example](../manifest/example/endpoint-secret.yaml)
 
+### Content-type
+You can specify the content type of the source image. The following content-type is valid:
+* kubevirt (Virtual disk image, the default if missing)
+* archive (Tar archive)
+If the content type is kubevirt, the source will be treated as a virtual disk and converted and sized appropriately. If the content type is archive it will be treated as a tar archive and CDI will attempt to extract the contents of that archive into the Data Volume.
```

Comment:
and converted to raw... (in case of qcow2 format)

Reply:
done